### DISCUSSION TITLE
Cut a release, pretty please? :)

### DISCUSSION THREAD

**Discussion by runiq** [ouch-org/ouch#254] [General]
Heya,

thanks for ouch :) Currently using it as a daily driver here and it's great. I'd really love to have https://github.com/ouch-org/ouch/pull/209 in a released version, since that is the only thing I was missing from 0.3.1.

Thank you for your consideration :)

**Comment by runiq** [comment]
Polite bump :)

**Comment by figsoda** [comment]
0.4.0 is published now

**Reply by marcospb19** [reply]
Yep, @runiq [0.4.0](https://github.com/ouch-org/ouch/releases) is finally here! :tada::tada::tada:

Took us some time, but the good news is that now we also have enough automation to easily make new releases.

**Reply by runiq** [reply]
That's awesome, thank both of you so much! <3
